code/game/turfs/simulated/floor/plating.dm

```diff
@@ -270,6 +270,11 @@
 	if (AM)
 		thing_to_check = list(AM)
 	for(var/thing in thing_to_check)
+		if(ismob(thing))
+			if(ishuman(thing))
+				var/mob/living/carbon/human/T = thing
+				if(FLYING in T.dna.species.specflags)
```

Comment:
so do you want wings to specifically get burned by it or all things to get burned because I really want to jetpack over lava with something that normally costs as much as a combat mech :^)